depscan/lib/utils.py

```diff
@@ -413,22 +413,29 @@ def export_pdf(
 
 
 def render_template_report(
-    jsonl_report_file,
+    vdr_file,
     summary,
     template_file,
     result_file,
 ):
     """
     Render the given json_report_file and summary dict using the template_file with Jinja
     """
-    with open(jsonl_report_file, "r", encoding="utf-8") as jsonl_file:
-        json_report = [json.loads(jline) for jline in jsonl_file.readlines()]
+    if vdr_file and os.path.isfile(vdr_file):
+        with open(vdr_file, "r", encoding="utf-8") as f:
+            bom = json.load(f)
+    else:
+        bom = None
```

Comment:
@heubeck Here you could fall back to loading the bom_file - just add the same thing you have for vdr_file in the else block.

Reply:
great idea, thx. will do that